startsida/index.html

```diff
@@ -0,0 +1,571 @@
+                        <br />kurspaket inom yrkeshögskolan. Myndigheten 
+                        <br />kommer att...</p>
+                </div>
+            </div>
+
+            
+            <div class="box box7">
+                <div class="Marknad">
+                    <svg class="rightpic" xmlns="http://www.w3.org/2000/svg" width="123" height="160" viewBox="0 0 123 160">
+                        <rect data-name="Rectangle 151" width="123" height="160" fill="#e3e3e3"/>
+                      </svg>
+                    
+                    <a href="#"><h2>Webbinarium :<br />
+                        Studerandestöd</h2></a>
+                        <br /><p>Myndigheten för yrkeshögskolan (MYH) har fått in 
+                        <br />fler ansökningar om att bedriva YH-utbildning än 
+                        <br />någonsin tidigare. De 1482 ansökningarna innebär 
+                        <br />en 35 procentig ökning jämfört med i fjol.</p>
+                </div>
+            </div>
+
+
+            <div class="box box8">
+                <div class="Marknad">
+                    <svg class="rightpic" xmlns="http://www.w3.org/2000/svg" width="123" height="160" viewBox="0 0 123 160">
+                        <rect data-name="Rectangle 151" width="123" height="160" fill="#e3e3e3"/>
+                      </svg>
+                    
+                    <a href="#"><h2>Rekordmånga ansökningar om att 
+                        <br />bedriva YH-utbildning</h2></a>
+                        <br /><p>Hur utformar vi ett bra studerandestöd när vi inte 
+                        <br />möts på plats? Omställningen till distansundervisning 
+                        <br />har skapat nya utmaningar när det gäller studerandestöd. 
+                        <br />De som tidigare inte har haft svårt att...</p>
+                </div>
+            </div>
+            
+            
+            <div class="box box9">
+             
+                        <div class="dots-carousel">
+                            <a href="#">
+                                <span class="controls" id="pause">&#x23f8;</span>
+                            </a>
+                            <span class="dot" onclick="currentSlide(1)"></span> 
+                            <span class="dot" onclick="currentSlide(2)"></span> 
+                            <span class="dot" onclick="currentSlide(3)"></span>
+                            <span class="dot" onclick="currentSlide(4)"></span> 
+                        </div>
+                        <div class="carousel-wrapper">
+                                
+                                <div class="mySlides fade">
+                                    <a href="/Nyhetsrum/Nyheter/2020/Fragor-och-svar-for-dig-som-utbildningsanordnare-med-anledning-av-Coronaviruset/">
+                                        <img src="images/GDHarOrdet_budget.JPG" alt="Nyheter" style="width:100%">
```

Comment:
Can't load images, getting a 404 not found error. Happens in both Chrome and Firefox, the issue seems to be related to uppercase image extension. Changing it to lowercase seems to fix the issue for me.

The problem is found in these lines; 318, 327, 336 and 345.